src/renderer/components/about-app.tsx

```diff
@@ -134,16 +134,7 @@ export default class AboutApp extends React.Component<{}, IState> {
               {i18n.t('Desktop Application', ABOUT_SYMPHONY_NAMESPACE)()}
             </span>
           </div>
-          <section>
-            <ul className='AboutApp-symphony-section'>
-              {symphonySectionItems.map((item, key) => (
-                <li key={key}>
-                  <strong>{item.key}</strong>
-                  <span>{item.value}</span>
-                </li>
-              ))}
-            </ul>
-          </section>
+          {sdaVersion ? this.renderVersions(symphonySectionItems) : null}
```

Comment:
We can just skip the null value, if it has value then this render things else nothing happen